libs/feature/record/src/lib/state/mdview.facade.spec.ts

```diff
@@ -239,6 +239,30 @@ describe('MdViewFacade', () => {
   })
 
   describe('geoDataLinksWithGeometry$', () => {
+    const links = [
+      {
+        type: 'download',
+        url: new URL('http://my-org.net/download/2.geojson'),
+        mimeType: 'application/geo+json',
+        name: 'Direct download',
+      },
+      {
+        type: 'service',
+        url: new URL('https://my-org.net/wfs'),
+        accessServiceProtocol: 'wfs',
+        name: 'my:featuretype', // FIXME: same as identifier otherwise it will be lost in iso...
```

Comment:
Does that mean that we use `name` instead of `identifierInService` everywhere?

Reply:
Hum, good question, I don't know, just moved this mock to reuse it